### PR TITLE
I1681 remove template fields related code

### PR DESCRIPTION
**Pull request by r-peschke**
This one is based on https://github.com/OpenSlides/openslides-backend/pull/1672 and sets the test coverage back to 96.6%

### CODE REVIEW

**Comment by jsangmeister** [comment]
@r-peschke Please rebase before I can review

**Comment by r-peschke** [comment]
> @r-peschke Please rebase before I can review

@jsangmeister done

File: openslides_backend/action/actions/meeting/export_helper.py [hunk 1]
```diff
@@ -141,21 +140,8 @@ def add_users(
...
-            template_fields.append(
-                (
-                    struct_field := field.get_structured_field_name(meeting_id),
-                    field.get_template_field_name(),
-                )
-            )
-            fields.append(struct_field)
-        else:
-            fields.append(field.own_field_name)
+        fields.append(field.own_field_name)
```
Comment (jsangmeister) [inline openslides_backend/action/actions/meeting/export_helper.py:144]: You can use a list comprehension here now

**Review by jsangmeister — CHANGES_REQUESTED** [review]
Minor remark, rest looks very satisfying. Did not check for other places, but if any pop up in the future, we can just remove it then.

**Pull request merged**